### PR TITLE
Added popup notification to channel messages

### PR DESCRIPTION
**Pull request by mayor04**
Popup is triggered when the user is not currently in the channel, When the popup is clicked the user is taken to the channel the message was sent

https://user-images.githubusercontent.com/67462604/135709279-177c9efe-15e0-4443-a695-eb84c2c7b0b7.mp4

### CODE REVIEW

File: lib/services/centrifuge_service.dart [hunk 1]
```diff
@@ -65,10 +124,29 @@ class CentrifugeService with ReactiveServiceMixin {
...
+      return null;
+    }
+    subList[channelSocketId] = subscription;
   }
 
   void dispose() {
     messageStreamController.close();
   }
 }
+// 0:"user_id" -> "6146430f1a5607b13c00bc13"
```
Comment (pauleke65) [inline lib/services/centrifuge_service.dart:139]: Please remove this commented code, the event output is likely to be different events, this is only for channels message event

Comment (mayor04) [reply lib/services/centrifuge_service.dart]: Done

**Review by jeremiahseun — DISMISSED** [review]
Looks good to me

File: lib/ui/view/channel/channel_view/channel_page_viewmodel.dart [hunk 2]
```diff
@@ -62,42 +70,40 @@ class ChannelPageViewModel extends BaseViewModel {
     await _channelsApiService.joinChannel(channelId);
   }
 
-  void getChannelSocketId(String channelId) async {
-    final channelSockId =
-    await _channelsApiService.getChannelSocketId(channelId);
+  // void getChannelSocketId(String channelId) async {
```
Comment (pauleke65) [inline lib/ui/view/channel/channel_view/channel_page_viewmodel.dart:73]: Delete commented code

**Review by pauleke65 — CHANGES_REQUESTED** [review]
Changes Requested

File: lib/ui/view/channel/channel_view/channel_page_viewmodel.dart (hunk 2 above)
Comment (mayor04) [reply lib/ui/view/channel/channel_view/channel_page_viewmodel.dart]: Done

**Review by pauleke65 — APPROVED** [review]

**Review by jeremiahseun — APPROVED** [review]
LGTM

**Pull request merged**